syntax = "proto3";

import "grpcdemo/proto/common.proto";

package greeter;

service Greeter { rpc SayHello(HelloRequest) returns (HelloResponse); }

message HelloRequest {
  string name = 1;
  common.Config config = 2;
}

message HelloResponse { string message = 1; }
